components/atom/tooltip/src/index.scss

```diff
@@ -0,0 +1,147 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$fz-atom-tooltip: $fz-m !default;
+$lh-atom-tooltip: $lh-m !default;
+$wlc-atom-tooltip: 4 !default; // -webkit-line-clamp → lines to show
+
+$maw-atom-tooltip: 256px !default;
+$mah-atom-tooltip: 88px !default;
+$c-atom-tooltip: $c-white !default;
+$bgc-atom-tooltip: $c-black !default;
+$bdrs-atom-tooltip: 4px !default;
```

Comment:
please create a new [sui-theme](https://github.com/SUI-Components/sui-theme/blob/master/src/settings/_box-style.scss) border radius var for this 4px

Don't worry in case the current border radius should be m sized or not, we can do naming refactor later on